experimental/packages/api-logs/src/api/logs.ts

```diff
@@ -48,6 +51,7 @@ export class LogsAPI {
       provider,
       NOOP_LOGGER_PROVIDER
     );
+    this._proxyLoggerProvider.setDelegate(provider);
 
     return provider;
```

Comment:
Aside: I was comparing this `setGlobal...` to the equivalent for the TracerProvider and noticed that the `internal/global-utils.ts` and global registration handling is very different in api-logs from what it is in the "api" package.  Apparently the code here was copied from api-metrics when that was a separate package ... but *before* the api-metrics package's global-registration handling was changed to https://github.com/open-telemetry/opentelemetry-js/pull/3357

When the api-logs package is merged into the "api" package, it will have to adapt to this new global-utils handling. That *could* be done before or after this PR.  Until then, the global registration handling is subtley different between them.

(`api-events` will want to change as well)

Reply:
Yeah I was planning to take care of that with api-logs migration to api package PR but that one got complicated, I will take care of using same global utils in different PR.